ambry-router/src/main/java/com.github.ambry.router/DecryptJob.java

```diff
@@ -23,7 +23,8 @@
  */
 class DecryptJob implements CryptoJob {
   private final BlobId blobId;
-  private final ByteBuffer enryptedContent;
+  private final ByteBuffer encryptedBlobContent;
+  private final ByteBuffer encryptedUserMetadata;
   private final ByteBuffer encryptedKey;
```

Comment:
can this be called `encryptedPerBlobKey` to be consistent with rest of the usage?